src/types/section.ts

```diff
@@ -44,6 +44,8 @@ export type ISection = {
   sectionType: SectionType;
   header: string;
   youtubeVideoUrl?: string;
+  title?:string;
+  slug?: string;
```

Comment:
@swornima-leapfrog 
isn't the title a mandatory property here? 

I believe title and slug should always exist and non empty string. Feel free to share your thoughts on this.

```suggestion
  title:string;
  slug: string;
```

Reply:
No @mrprayaspoudel  dai, here in section we are expecting data from two different contentful model. Where one has slug and title and another doesnot. So we cannot keep the property mandatory.